virtcontainers/clh.go

```diff
@@ -122,6 +122,18 @@ var clhDebugKernelParams = []Param{
 //
 //###########################################################
 
+func (clh *cloudHypervisor) checkVersion() error {
+	if clh.version.Major > supportedMajorVersion || (clh.version.Major == supportedMajorVersion && clh.version.Minor > supportedMinorVersion) {
```

Comment:
what is the version compatibility policy of clh? Why are we not filtering minimal supported version (as implemented originally in clh.go)?

Reply:
/cc @sboeuf.

Reply:
@bergwolf 
w.r.t. 'filtering minimal supported version', I think you were referring to some code which is not currently in clh.go

Reply:
We don't really have any policy yet regarding the compatibility of different versions. The project just released `0.4.0` but we're not yet at `1.0`.
/cc @sameo

Reply:
@tedyu I was thinking something like:
```
if clh.version.Major > supportedMajorVersion || (clh.version.Major == supportedMajorVersion && clh.version.Minor >= supportedMinorVersion) {
    // proceed
} else {
   // fail
}
```
Then we can define `supportedMajor/MinorVersion` as minimal supported version instead of maximum supported version.

Reply:
In that case, currently minimum supportedMajorVersion would be 0.
What value would minimum supportedMinorVersion be (3) ?

Reply:
The code is to implement mechanism. We can bump minimal supported version when needed.

Reply:
@bergwolf 
I have addressed your comment.